pennylane/qnodes/base.py

```diff
@@ -457,14 +471,15 @@ def _construct(self, args, kwargs):
         converts it into a circuit graph, and creates the VariableRef mapping.
 
         .. note::
-           The VariableRefs are only required for analytic differentiation,
-           for evaluation we could simply reconstruct the circuit each time.
+           In mutable circuits the VariableRefs only represent positional args,
```

Comment:
This part of the sentence doesn't seem to align with the description given on lines 481-482

Reply:
I mean to say that VariableRefs are used to represent the elements of positional args. I'll reword it so it's clear.